test/fixtures/parser/ansi/select_cross_join.yml

```diff
@@ -0,0 +1,72 @@
+file:
+  statement:
+    with_compound_statement:
+      keyword: WITH
+      common_table_expression:
+        identifier: constants
+        keyword: AS
+        start_bracket: (
+        select_statement:
+          select_clause:
+            keyword: SELECT
+            select_target_element:
+              literal: '8760'
+              alias_expression:
+                keyword: AS
+                identifier: hours_per_year
+        end_bracket: )
+      select_statement:
+        select_clause:
+        - keyword: SELECT
+        - select_target_element:
+            column_reference:
+            - identifier: table1
+            - dot: .
+            - identifier: name
+        - comma: ','
+        - select_target_element:
+            column_reference:
+            - identifier: foo
+            - dot: .
+            - identifier: name
+        - comma: ','
+        - select_target_element:
+            expression:
+            - column_reference:
+              - identifier: foo
+              - dot: .
+              - identifier: value
+            - binary_operator: '*'
+            - column_reference:
+              - identifier: constants
+              - dot: .
+              - identifier: hours_per_year
+            alias_expression:
+              keyword: AS
+              identifier: some_value
+        from_clause:
+        - keyword: FROM
+        - table_expression:
+            main_table_expression:
+              table_reference:
+                identifier: table1
+        - join_clause:

```

Comment:
The parse tree looks good in the sense that the two joins are at the same level. I'll defer to you and Niall about the question he raised.